app/(tabs)/home.tsx

```diff
@@ -1,10 +1,81 @@
-import { Text, View } from 'react-native';
+import {
+  Text,
+  View,
+  FlatList,
+  ActivityIndicator,
+  RefreshControl,
+  Dimensions,
+} from 'react-native';
+import { useState, useCallback } from 'react';
+import { Card } from '../../components/Card';
+
+// Simulamos datos de ejemplo
+const generateMockData = (page: number) => {
+  return Array(10)
+    .fill(0)
+    .map((_, index) => ({
```

Comment:
This looks a bit weird, can we do a regular for loop with a type for the entries please? Can you also add a return type to the function?